app/views/organisations/_show_organisation.html.erb

```diff
@@ -1,9 +1,33 @@
-<div class="organisation-logo">
-  <%= organisation.logo["formatted_title"] %>
-</div>
+<% content_for :breadcrumbs do %>
+  <div class="margin-bottom">
+    <% if @header.is_no_10? %>
+      <h1 class="no10-banner margin-bottom"><%= @header.org.title %></h1>
+    <% else %>
+      <h1 class="visually-hidden"><%= @header.org.title %></h1>
+    <% end %>
+  </div>
+<% end %>
+
+<% unless @header.is_no_10? %>
+  <div class="grid-row">
+    <div class="<%= @header.logo_wrapper_class %> margin-bottom">
+      <%= render "govuk_publishing_components/components/organisation_logo", {
+        organisation: @header.organisation_logo
+      } %>
+    </div>
 
-<%= render 'featured_links',
-           featured_links: organisation.ordered_featured_links %>
+    <div class="<%= @header.link_wrapper_class %> margin-bottom">
+      <%= render "govuk_publishing_components/components/translation-nav", @header.translation_links %>
+      <%= render "components/topic-list", @header.ordered_featured_links %>
+    </div>
+  </div>
+<% end %>
+
+<% if false %>
```

Comment:
I would still be tempted to remove this and the code beneath as we can always refer back to the Git history for this, but up to you 🙂

Reply:
I intend to address this in the very next card, the news section.